Week01/Problem05/cyu_05.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+"""This script is written by Chuanping Yu, on Jul 24, 2017,
+for the Assignment#1 in IDEaS workshop"""
+
+#Problem 5
+from fractions import gcd
```

Comment:
Good use of existing modules!  Doing this in the future may save you hours, if not days, of time.